### PR TITLE
Feat/preview uploaded picture

### PR DESCRIPTION
**Pull request by yvlange**
With this PR the user is able to preview the image she/he wants to upload to the trip.
The changes are made in AddForm.js and EditFrom.js. 
In EditForm the alt text of the image is visible although if no image is selected yet, the 'preview' text should not be visible. 
If anyone knows why that is, I'd appreciate some hints 😊 
Thank you for your review! 🚀

### CODE REVIEW

**Review by soerendolling — APPROVED** [review]
Nice Work Yvonne. I saw When the fileName is really long the name is conflicting with your layout. May be it can be wrapped. I can´t think of something why the Alt text is displayed

File: src/components/UploadPhoto.js [hunk 1]
```diff
@@ -1,19 +1,15 @@
...
   return (
     <div className="upload">
-      {/* <label for="file" className="upload__label">
-        <BsUpload className="upload__icon" />
-        Choose a Photo
-      </label> */}
       <input
         type="file"
+        accept="image/*"
+        multiple
```
Comment (marwinburesch) [inline src/components/UploadPhoto.js]: I thought you decided to disable multiple image upload for now :)
If so I'd advise to remove this option

File: src/components/EditTrip.js [hunk 2]
```diff
@@ -98,13 +103,24 @@ function EditTrip() {
             setNotesInput(e.target.value);
           }}
         />
+
         <UploadPhoto
           id="photo"
           name="photo"
           onChange={(e) => {
             setImageUpload(e.target.files[0]);
+
```
Comment (marwinburesch) [inline src/components/EditTrip.js:111]: This looks quite different to what I saw in `src/components/AddForm.js` line 95

File: src/components/EditTrip.js [hunk 3]
```diff
@@ -55,6 +58,8 @@ function EditTrip() {
     setImageUpload(myTrip.photo);
   }, [id]);
 
+  console.log(imageUpload);
```
Comment (marwinburesch) [inline src/components/EditTrip.js]: Oh, a stray `console.log()` 😈

**Review by marwinburesch — CHANGES_REQUESTED** [review]
Looking good, nice work! 🐜 
Just a few questions

**Pull request merged**